en/index.md

```diff
@@ -7,6 +7,20 @@ for Canonical documentation projects. Topics are listed in the navigation
 to the left, and presented here as a single page to aid searching.
 
 
+## Diátaxis
+
+Use Diátaxis to structure product documentation. A documentation page should
+fit within one of the following categories:
+
+- Tutorial
+- How-to
+- Explanation
+- Reference
+
+Each category serves a different user need and is written in a specific style.
```

Comment:
are they really written in a 'specific style'? I think that could be misinterpreted. My more fuzzy suggestion
```suggestion
Each category serves a different user need - it is useful to understand the needs being addressed as these may influence the documentation approach. 
```